packages/manager/modules/sms/src/sms/sms/incoming/telecom-sms-sms-incoming.html

```diff
@@ -12,31 +12,17 @@ <h1 data-translate="sms_sms_incoming_title"></h1>
         <tuc-toast-message></tuc-toast-message>
 
         <div class="mb-3 clearfix">
-            <div
-                class="btn-group float-left mr-3"
-                data-uib-dropdown
-                data-dropdown-append-to-body
-            >
+            <div class="btn-group float-left mr-3" data-uib-dropdown>
```

Comment:
Maybe it could be preferable to use directly a `oui-dropdown` instead of the bootstrap component :)

Reply:
You're right!

It requires a large refactoring to drop `uib-dropdown` but also `angular-boostrap`.
The following project has been be paused for now but yes it is still planned.

Stay tuned!